AElf.Kernel/Block.cs

```diff
@@ -1,16 +1,12 @@
-﻿using Newtonsoft.Json;
-using System;
-using System.IO;
-using System.Runtime.Serialization;
-using System.Runtime.Serialization.Formatters.Binary;
-using System.Security.Cryptography;
-using System.Text;
+﻿using System;
+using System.Collections.Generic;
 
 namespace AElf.Kernel
 {
     [Serializable]
     public class Block : IBlock
     {
+        #region Seems useless for now, but maybe useful in the future
```

Comment:
If the the properties are not used and seem useless, delete them, keep the code clean